config_management/group_vars/all

```diff
@@ -2,8 +2,9 @@
 go_version: 1.7.4
 terraform_version: 0.8.5
 docker_version: 1.11.2
-kubernetes_version: 1.5.2
-kubernetes_cni_version: 0.3.0.1
+docker_install_role: 'docker-from-get.docker.com'
+kubernetes_version: 1.5.6
```

Comment:
Looks like it actually doesn't, well spotted.
Ansible was stuck, and when running manually I get an infinite loop of:
```
[apiclient] First node has registered, but is not ready yet
```
I'll move this branch back to K8S 1.5.2 as it seems to still be in the APT repository now, and CNI 0.3.0.1, until this gets resolved on `kubeadm`'s side.